style.css

```diff
@@ -0,0 +1,186 @@
+/* Styles reset */
+* {
+	box-sizing: border-box;
+}
+
+body {
+	font-size: 14px;
+	margin: 0;
+}
+
+h2, h3, h4 {
+	margin: 0;
+}
+
+
+ul {
+	list-style-type: none;
+	padding: 0;
+	margin: 0;
+}
+
+input,
+select {
+	border: 0;
+	margin: 0;
+}
+
+.wrapper {
+	margin: 0 auto;
+	width: 960px;
+}
+
+.home {
+	background: url(images/home-bg.jpg) no-repeat;
+	background-size: cover;
+	padding: 85px 0;
+	min-height: 852px;
+}
+
+.header {
+	margin-bottom: 218px;
+}
+
+.logo {
+	display: inline-block;
+    font-size: 0;
+	margin-top: 4px;
+}
+	
+.navigation {
+	float: right;
+}
+
+.navigation li {
+	float: left;
+	margin-left: 25px;
+
+}
+
+.nav-link {
+	color: #ffffff;
+	text-decoration: none;
+	text-transform: uppercase;
+	display: inline-block;
+	border-top: 4px solid transparent;
+	padding: 10px 0;
+}
+
+.nav-link:hover {
+	color: #d6d6d6;
+}
+
+.nav-link.active {
+	border-top-color: #4acaff;
+}
+
+.job-search-form-header {
+	font-size: 55px;
+	color: #FFFFFF;
+	font-style: italic;
+	font-weight: normal;
+	margin-bottom: 40px;
+}
+
+.job-search-form-col-1 {
+	width: 76%;
+	float: left;
+}
+
+.job-description-input {
+	width: 100%;
+	height: 54px;
+	border-bottom: 4px solid #e2e2e2;
+	font-size: 18px;
+	padding: 0 20px;
+	line-height: 54px;
+	margin-bottom: 30px;
+}
+
+.job-description-input::-webkit-input-placeholder {
+	color: #c7c7c7;
+	font-style: italic;
+}
+
+.job-search-form-checkbox {
+    margin-right: 15px;
+}
+
+.job-search-form-label {
+	color: #f6f6f6;
+	font-style: italic;
+	margin-right: 37px;
+}
+
+.user-location {
+	float: right;
+	height: 37px;
+    width: 247px;
+    padding: 0 14px;
+}
+
+.job-search-form-col-2 {
+	width: 15%;
+	float: right;
+}
+
+.job-search-form-submit-button {
+	background: #4acaff url(images/icon-search.png) 28px center no-repeat;
+	padding: 0 10px 0 50px;
+	color: #ffffff;
+	font-size: 20px;
+	height: 54px;
+	width: 100%;
+	border-bottom: 4px solid #3baedd;
+	font-style: italic;
+	text-align: left;
+	cursor: pointer;
+}
+
+.about-us {
+	background-color: #f5f5f5;
+	padding: 107px 0;
+	min-height: 590px;
+}
+
+.about-us h4 {
+	font-size: 30px;
+	color: #585858;
+	text-align: center;
+    padding-bottom: 16px;
+}
+
+.about-us h3 {
+	font-size: 45px;
+	color: #585858;
+	text-align: center;
+	padding-bottom: 85px;
+}
+
+.about-us-top {
+	font-size: 18px;
+	color: #585858;
+	text-align: center;
+}
+.about-us-bottom {
+	font-size: 18px;
+	color: #585858;
+	text-align: center;
+	margin: 0 auto 60px auto;
+}
```

Comment:
.about-us-top and .about-us-bottom - please change back text to 1 p (not two separate p's to move text to next row)...to move text to a second row just set width to div (measure from psd), and add margin: 0 auto